### ISSUE TITLE
Changes requested from Fundamental-* libraries 

### ISSUE THREAD

**Issue by jbadan** [SAP/fundamental#1338]
| Topic | Status | Notes |
|-------|--------|-------|
| Fixed height calendars | _Pending_ | See https://github.com/SAP/fundamental-react/pull/396 |
| Making tabs buttons (instead of anchors) | #1354 | See [WAI Spec for Tabs](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html#rps_label) |
| Using SVG icons instead of icon font | _Pending_ | See https://css-tricks.com/icon-fonts-vs-svg/ and https://github.com/jonathantneal/svg4everybody |
| Use of "Tile as a Button" | #1348 | See https://sap.github.io/fundamental/components/tile.html#tile-as-a-button. If it's a button, then it should be a button styled as a tile, not a `<div>` with a `role="button"`. |
| Token component with role='button' | #1346  | Should be `<button>` instead of `span role='button>` |
| Popover component | This should be solved by the updates post switch to popper library #1318  | replace `<div className='fd-popover__control'>` with `<button>` |
| Menu component | #1349 | `<ul>` add `role='menu`, `<li>` add `role='presentation`, `<a>` add `role='menuitem` and `tabIndex='-1'` |
| Calendar component |#1350 | months and years lists: `<span>` tag to become a `<button>`. Table cells should have `role='cell`' instead of `role='gridcell'` because they are clickable |
| Icon component | #1351| per "Icons can be used adaptively if desired, but at this point they are used more as visual elements within other components", we need to be able to render both a` <span>` and `<button>`. When rendering a `<span>` a `role='image'` should be added along with `alt='{icon-description}'` |
| Shellbar Component | Need to re-evaluate | Logo & LogoSAP: `<a>` tag to become a `<button>`, nav-back: `<span>` becomes `<button>`, productSwitcherList: `<a>` tag to become a `<button>` |
| Pagination component |#1352 | `<a>` tags should be `<button>` |
| SearchInput component | _Pending_ | should be combobox component |
| MultiInput component | #1353 | `<span role='button'>` should be `<button>` element |
| Form element construction | _Pending_ | See [Gist](https://gist.github.com/ChristianKienle/619782c76b0f3165da770aa93018c79b#form-items-require-special-class-depending-on-the-type-of-input-elements) for details |
| What's the difference between `FormSet` and `FormGroup`? | _Pending_ | |
| Buttons with just icons need `aria-label` | _Pending_ | |
| Color contrast issues | [#1297](https://github.com/SAP/fundamental/issues/1297) | Links and Medium Buttons do not meet color contrast standards |
| Popover styling | [#1286](https://github.com/SAP/fundamental/issues/1286) | Styling changes needed for using popper.js. Box shadows should be reworked so components can "flip". The markup for a "popper" overlay is also different. |
| Use of sass to build full class names | Addressed during fundamental-sync  | Should the `*.scss` files leverage sass's ability to concatenate strings to build full class names or should each block have the full class name? See the [SASS Syntax](https://github.com/SAP/fundamental-react/wiki/Topics-for-Fiori-Fundamentals#sass-syntax) example below. |
| ActionBar Component Back Buttons needs `aria-label` | [#1360](https://github.com/SAP/fundamental/issues/1360) |  |
| Investigate moving away from Travis | [#499](https://github.com/SAP/fundamental-react/issues/499) | Need to align on which option is the best |

**Comment by ChristianKienle** [comment]
I would love to also discuss this #1346. It actually has been requested by a team evaluating Fundamentals right now.

**Comment by ChristianKienle** [comment]
I think this was never made it to the list: 
https://gist.github.com/ChristianKienle/93d18c939010a31db07be940d66bd728#redundancy-when-using-button-groups
